fastlane/Fastfile

```diff
@@ -25,7 +25,7 @@ files_with_version_number = {
     './ios/Classes/PurchasesFlutterPlugin.m' => ['return @"{x}"'],
     './android/build.gradle' => ['version \'{x}\''],
     './android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java' => ['PLUGIN_VERSION = "{x}"'],
-    './purchases_ui_flutter/pubspec.yaml' => ['version: {x}'],
+    './purchases_ui_flutter/pubspec.yaml' => ['version: {x}', 'purchases_flutter: ^{x}'],
```

Comment:
I guess if we are doing this, we could specify the specific package (which could probably be a good thing IMO...

Reply:
do you mean the specific package version? I got a warning when doing that:
```
* Your dependency on "purchases_flutter" should allow more than one version. For example:
  
  dependencies:
    purchases_flutter: ^6.22.0
  
  Constraints that are too tight will make it difficult for people to use your package
  along with other packages that also depend on "purchases_flutter".
```

should I add the specific version and ignore that?

Reply:
Oh sorry! Yeah I meant that... Basically so people don't accidentally use a mismatched version of purchases_flutter and purchases_ui_flutter

Reply:
perfect I did that. Makes sense. We will get a warning when publishing but it's fine